src/Tribe/RSVP.php

```diff
@@ -655,7 +655,15 @@ public function generate_tickets( ) {
 		}
 	}
 
-	public function send_tickets_email( $order_id ) {
+	/**
+	 * Dispatches a confirmation email that acknowledges the user has RSVP'd
+	 * including the tickets.
+	 *
+	 * @param int $order_id
+	 * @param int $event_id
+	 * @since TBD added $event_id parameter
```

Comment:
Don't need to change this, but the ideal grouping for future ref is meta tags like `@since` first then a break with `@param` tags next of all then another break with the `@return` tag if needed, ie:

```
/**
 * @since x.y.z
 * @users other_func()
 *
 * @param int $foo
 * @param int $bar
 * 
 * @return foobar
 */
```